README.md

```diff
@@ -71,22 +71,22 @@ If you want to build with a different compiler, pass the `CC` and `CXX` environm
 
 #### Note on installing CUDA on Ubuntu
 
-Nvidia provides .deb packages. CUDA will be installed in `/usr/local/cuda-10.0` and requires 3GB of diskspace.
+NVIDIA provides .deb packages. CUDA will be installed in `/usr/local/cuda-10.0` and requires 3GB of disk space.
 If your `/usr/local` partition doesn't have that much space left you can create a symbolic link before
 doing the install; for example: `sudo ln -s /opt/cuda-10.0 /usr/local/cuda-10.0`
 
-The instructions given on the nvidia website tell you to finish with `apt install cuda`. However, this
+The instructions given on the NVIDIA website tell you to finish with `apt install cuda`. However, this
 might not work (missing dependencies). In that case use `apt install cuda-10-0`. Afterwards you can
 install the meta package `cuda` which will cause an automatic upgrade to a newer version when that
 comes available (assuming you use `Installer Type deb (network)`, if you'd want that (just cuda-10-0 will
 stay at version 10). If you don't know what to do, only install cuda-10-0.
 
 cuDNN exists of two packages, the Runtime Library and the Developer Library (both a .deb package).
 
-Before you can download the latter you need to create a (free) "developer" account with nvidia for
-which at least a legit email address is required (their website says: The e-mail address is not made public
+Before you can download the latter you need to create a (free) "developer" account with NVIDIA for
```

Comment:
There should probably be a comma after "latter".

Reply:
Already added.